projects/canopy/src/lib/forms/checkbox-group/checkbox-group.module.ts

```diff
@@ -10,6 +10,5 @@ import { LgCheckboxGroupComponent } from './checkbox-group.component';
   imports: [ LgLabelModule, LgMarginModule, LgFocusModule ],
   declarations: [ LgCheckboxGroupComponent ],
   exports: [ LgCheckboxGroupComponent ],
-  entryComponents: [ LgCheckboxGroupComponent ],
```

Comment:
Is this no longer a thing?

Reply:
Doesn't look like it - it was automatically removed!

Reply:
> With Angular 9 coming in and Ivy as the new rendering engine, all the components would be considered as entering components and do not necessarily need to be specified inside the entryComponents array.